src/python/gem5/components/processors/abstract_core.py

```diff
@@ -66,7 +66,6 @@ def connect_icache(self, port: Port) -> None:
         """
         This function should connect the response port from the instruction
         cache to the right request port on the core.
-
```

Comment:
I think there's some weird formatting here where these new lines have been removed. Can you re-add them?